### PR TITLE
Issue-359: Added token designator to the field name entered into the …

### PR DESCRIPTION
**Pull request by friedlou**
…metadata table for tokenized fields.

### CODE REVIEW

File: warehouse/ingest-core/src/test/java/datawave/TestAbstarctContentIngestHelper.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package datawave;
+
+import com.google.common.collect.Multimap;
+import datawave.ingest.data.RawRecordContainer;
+import datawave.ingest.data.config.NormalizedContentInterface;
+import datawave.ingest.data.config.ingest.AbstractContentIngestHelper;
+
+public class TestAbstarctContentIngestHelper extends AbstractContentIngestHelper {
```
Comment (ivakegg) [inline warehouse/ingest-core/src/test/java/datawave/TestAbstarctContentIngestHelper.java]: Abstarct -> Abstract

File: warehouse/ingest-core/src/main/java/datawave/ingest/metadata/EventMetadata.java [hunk 2]
```diff
@@ -312,8 +312,8 @@ protected void addTokenizedContent(IngestHelperInterface helper, RawRecordContai
             for (String field : fields.keySet()) {
                 if (h.isContentIndexField(field)) {
                     updateForIndexedField(helper, event, fields, countDelta, loadDate, tokenDesignator, field);
-                    termFrequencyFieldsInfo.createOrUpdate(field, event.getDataType().outputName(), MetadataWithMostRecentDate.IGNORED_NORMALIZER_CLASS,
-                                    event.getDate());
+                    termFrequencyFieldsInfo.createOrUpdate(field + tokenDesignator, event.getDataType().outputName(),
```
Comment (ivakegg) [inline warehouse/ingest-core/src/main/java/datawave/ingest/metadata/EventMetadata.java]: Can tokenDesignator be null?

Comment (friedlou) [reply warehouse/ingest-core/src/main/java/datawave/ingest/metadata/EventMetadata.java]: Fixed in latest commit.

File: warehouse/ingest-core/src/test/java/datawave/TestAbstarctContentIngestHelper.java (hunk 1 above)
Comment (friedlou) [reply warehouse/ingest-core/src/test/java/datawave/TestAbstarctContentIngestHelper.java]: Fixed in latest commit.

**Pull request merged**